conf/urls.py

```diff
@@ -538,6 +538,16 @@
         prototype.views.NewsArticleDetailView.as_view(),
         name='news-article-detail',
     ),
+    url(
+        r"^international/news/$",
```

Comment:
`international/` or `int/`?

Reply:
international homepage is `international/` so `international/news/` makes sense here. i don't think we use `int/` anywhere